docs/software/cloud.md

```diff
@@ -60,19 +96,279 @@ Nextcloud gives you access to all your files wherever you are. Store your docume
     - 64bit systems can handle 8796 PB (petabytes)
 
-    **Will my data be saved after deinstallation?**
-    Your userdata directory will stay after deinstallation. As well a database backup will be saved to your userdata directory. Thus you can easily restore your instance by reinstalling Nextcloud and restore the database dump.
+    #### Will my data be saved after deinstallation?
+
+    Your user data directory will stay after deinstallation. As well a database backup will be saved to your user data directory. Thus you can easily restore your instance by reinstalling Nextcloud and restore the database dump.
+
+See also <https://nextcloud.com/athome>.
+
+## Nextcloud Talk - Video calls via Nextcloud, including TURN server
+
+Host video calls on your own Nextcloud instance. The TURN server ***coturn*** will be installed and configured as well to allow reliable video calls through outside the local network, NAT and firewall setups.
+
+Also installs:
+
+- Nextcloud
+- coturn
+
+![DietPi cloud software Nextcloud Talk](../assets/images/dietpi-software-cloud-nextcloudtalk.png)
+
+### Installation notes
+
+During installation you will be asked to enter the external server domain and a port, that you want to use for the coturn TURN server. Note that you need to forward the chosen port and/or open it in your firewall.
+
+If HTTPS was or is enabled via `dietpi-letsencrypt`, coturn will be configured to use the LetsEncrypt certificates for TLS connections on the chosen TURN server port automatically.  
+coturn by default will listen to non-TLS requests as well on the port configured in `/etc/turnserver.conf`. You can force TLS/control this by switching port forwarding in your router and/or opening/dropping ports in your firewall.
+
+coturn logging by default is disabled via `/etc/default/coturn` command arguments, since it is very verbose and produces much disk I/O. You can enable and configure logging via `/etc/turnserver.conf`, if required.
+
+## Pydio - Feature-rich backup and sync server with web interface
+
+Pydio is a feature-rich backup and sync server with web interface. Similar to ownCloud with vast configuration options to meet your "cloud" needs.
+
+Also Installs:
+
+- Webserver
+
+![DietPi cloud software Pydio](../assets/images/dietpi-software-cloud-pydio.png)
+
+=== "Access to the web interface"
+
+    URL = `http://<your.IP>/pydio`
+
+=== "First time connect"
+
+    - Ignore the warnings and click the button titled `CLICK HERE TO CONTINUE TO PYDIO`.  
+      Remark: If you require SSL access, please use LetsEncrypt to set this up.
+    - The wizard can now be started, click the `start wizard >` button to begin.  
+    - Enter and create a new admin account for use with Pydio. Then click the `>>` button.
+    - Under database details, enter the following:
+        - Database type = `MySQL`
+        - Host = `localhost`
+        - Database = `pydio`
+        - User = `pydio` (Previous to DietPi v6.0 = `root`)
```

Comment:
Let's not show ancient information anymore and remove `(Previous to DietPi v6.0 = `root`)`.

Reply:
Ok, removed it.